### PR TITLE
Add queryRenderedFeatures with bounding box around cursor API docs example

### PR DESCRIPTION
**Pull request by lucaswoj**
cc @1ec5 @jfirebaugh @lyzidiamond @danswick 

ref https://github.com/mapbox/mapbox-gl-js/issues/2537 

screen shot 2016-05-10 at 10 38 42 am

### CODE REVIEW

File: js/ui/map.js [hunk 1]
```diff
@@ -405,10 +405,20 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
...
      * @example
+     * // Find all features within a static bounding box
      * var features = map.queryRenderedFeatures([[10, 20], [30, 50]], { layers: ['my-layer-name'] });
+     *
+     * @example
+     * // Find all features within a bounding box around a point
+     * var width = 10;
+     * var features = map.queryRenderedFeatures([
+     *   [point.x - width / 2, point.y - width / 2],
+     *   [point.x + width / 2, point.y + width / 2]
```
Comment (1ec5) [inline js/ui/map.js]: There should be separate `width` and `height` variables, even if their values happen to be the same.

Comment (lucaswoj) [inline js/ui/map.js]: Fixed in latest commit

**Comment by mollymerp** [comment]
There might be confusion around units here – when I think height/width in web dev I'm usually thinking about pixels (or `em` etc) but in the last example in this PR width and height are degrees right? Also, the `radius` param in queryRenderedFeatures was in pixel units. 

Maybe add a note in a comment about the units?

**Comment by lucaswoj** [comment]
The units are pixels. This is explained in the documentation for the method (cut off in the screenshot above).

**Comment by mollymerp** [comment]
😳 Oops! Never mind then -- looks good!

**Pull request merged**